Tasks/ANT/task.json

```diff
@@ -9,10 +9,13 @@
         "Build",
         "Release"
     ],
+    "runsOn": [
+        "Agent"
+    ],
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 0,
+        "Minor": 1,
```

Comment:
I would avoid rev'ing all tasks just to add runsOn Agent.  It should default to that.  

Don't require the server to store and agents to redownload for no reason.